src/main/java/org/skriptlang/skript/bukkit/potion/elements/conditions/CondIsPotionInstant.java

```diff
@@ -0,0 +1,37 @@
+package org.skriptlang.skript.bukkit.potion.elements.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.potion.PotionEffectType;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+@Name("Potion Is Instant")
+@Description({
+	"Checks whether a potion effect type is instant.",
+	"That is, whether the effect happens once/immediately."
+})
+@Example("""
+	if any of the potion effects of the player's tool are instant:
+		message "Use your tool for immediate benefits!"
+""")
```

Comment:
All of these examples have an extra level of indentation because of where the ending """ are
```suggestion
	if any of the potion effects of the player's tool are instant:
		message "Use your tool for immediate benefits!"
	""")
```